src/_component/layout/about-me/about-me.pug

```diff
@@ -0,0 +1,14 @@
+.section-title Welcome to My Cv 
+button.disc
+    i.icon_youtube 
+    span.button-text YT 
+
+h1 Paul Nicolae Geambazu
+h2 Junior Web Developer 
+h3 About me 
+.desription-box__with-shadow 
```

Comment:
I do not know if you meant to put ‘.desription-....’, but I thought i should point out. Sorry 😬